### PR TITLE
sonarlint :: configutil cleanup

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

**Review by andrewbp — APPROVED** [review]
LGTM

File: src/main/java/emissary/config/ConfigUtil.java [hunk 1]
```diff
@@ -568,26 +562,23 @@ public static InputStream getConfigData(final String f) throws IOException {
...
             // sort the files, to put emissary.admin.MasterClassNames.cfg before emssary.admin.MasterClassNames-blah.cfg
-            Arrays.sort(files);
-            masterClassNames.addAll(Arrays.asList(files));
+            if (files != null) {
+                Arrays.sort(files);
+                masterClassNames.addAll(Arrays.asList(files));
+            }
         }
         // check to make sure we have at least one
         // TODO make a test for this
```
Comment (cfkoehler) [inline src/main/java/emissary/config/ConfigUtil.java:573]: Yep. We still need a test for this!

File: src/main/java/emissary/config/ConfigUtil.java [hunk 2]
```diff
@@ -104,16 +103,16 @@ public class ConfigUtil {
      * XML bean configuration with Jetty 6 does a poor job of propagating exceptions during configure. Once we update to
      * Jetty 9, evaluate removing this. Otherwise, there is no way I could find to stop the server from starting if there is
      * a configuration error.
-     * 
+     * <p>
      * TODO: evaluate whether we can change this now that we are on jetty 9
```
Comment (cfkoehler) [inline src/main/java/emissary/config/ConfigUtil.java:107]: Has this evaluation been done? Or is this still an open task? If so we should open an issue for it.

**Review by cfkoehler — APPROVED** [review]

File: src/main/java/emissary/config/ConfigUtil.java (hunk 2 above)
Comment (dev-mlb) [reply src/main/java/emissary/config/ConfigUtil.java:107]: https://github.com/NationalSecurityAgency/burrito-grande/issues/1919

File: src/main/java/emissary/config/ConfigUtil.java (hunk 1 above)
Comment (dev-mlb) [reply src/main/java/emissary/config/ConfigUtil.java:573]: https://github.com/NationalSecurityAgency/burrito-grande/issues/1920

**Pull request merged**